src/include/txn_inline.h

```diff
@@ -1132,14 +1125,24 @@ __wt_txn_read(
             if (cbt->upd_value->skip_buf) {
                 cbt->upd_value->buf.data = NULL;
                 cbt->upd_value->buf.size = 0;
+            } else if (ovfl_retry && is_ovfl_rm) {
```

Comment:
Will this be reached? If the cell wasn't unpacked, will the `tw` be populated correctly so that the start_visible check is executed?

Reply:
Yes, in the code, even we return WT_RESTART, the cell is unpacked correctly.

Reply:
Fixed